qcfractal/cli/tests/test_cli.py

```diff
@@ -34,7 +34,7 @@ def active_server(request):
     with testing.popen(args, **_options) as server:
         time.sleep(2)
 
-        server.test_uri_cli = "--fractal-uri=localhost:" + port
+        server.test_uri_cli = "--fractal-address=localhost:" + port
```

Comment:
This is a URI not an address, please revert.

Reply:
Would it be better to call the `FractalClient`' kwarg `uri` instead of `address`? This would help make the term more uniform if we want to be strict about calling it `uri`.

If thats the case, would you accept the alias of `address`?

Reply:
`FractalClient` can be a URI or address, this *must* be a URI.